README.md

```diff
@@ -5,41 +5,87 @@
   </a>
 </p>
 
-# Docker Image
-    
-**1. Run a container with PHPSchool.**
-  
+# Getting Started
 
-```bash
+This process will provide you with a Docker environment where you can quickly run PHP School Workshops. We've gone ahead and pre-installed a few for you such as `learnyouphp`!
 
-  docker run -i -t -v ~/Dropbox/docker-phpschool:/phpschool/code:rw -e "PATH=/root/.php-school/bin:$PATH" rafaelcgstz/phpschool bash
+Once you get it up and running you should complete your tasks inside the `code` directory and use an editor such as [Atom](https://atom.io/) to write them.
 
-```
+Everything in the `code` directory will be mounted inside the Docker container at `/phpschool/code` so you can navigate there using the `cd` command and verify your solutions!
```

Comment:
I'm working on the assumption most if not all people will be using an IDE to create files and code. I'll add a note of this and example of how they would verify in a given example.